frontend/src/App.jsx

```diff
@@ -0,0 +1,138 @@
+import React, { useState, useEffect } from "react";
+import AudioPlayer from "react-h5-audio-player";
+import "react-h5-audio-player/lib/styles.css";
+import "./App.css";
+import axios from "axios";
+
+const App = () => {
+	const [data, setData] = useState(null);
```

Comment:
create a separate function for this and call under app.
It would be simpler to read.